src/pages/docs/ai-transport/features/sessions-identity/resuming-sessions.mdx

```diff
@@ -0,0 +1,140 @@
+- **Channel-based addressing**: The agent only needs the channel name to reconnect, no need to track individual client connections or manage connection state
+
+### Catching up on messages using history
+
+When an agent resumes, it needs to retrieve messages published while it was offline. Use [channel history](/docs/storage-history/history) with the [`untilAttach` option](/docs/storage-history/history#continuous-history) to catch up on historical messages while preserving continuity with live message delivery.
+
+<Aside data-type="note">
+Clients require the [`history`](/docs/auth/capabilities) capability on the channel to retrieve message history.
+</Aside>
+
+#### Persisted session state
+
+Your agent should persist the following state to enable resumption:
+
+- **Channel name**: The channel the agent was processing
+- **Last processed timestamp**: The timestamp of the last message successfully processed by the agent
+
+This state allows the agent to reconnect to the correct channel and retrieve only the messages it missed.
+
+#### Catching up with continuity
+
+The recommended pattern uses `untilAttach` to paginate backwards through history while maintaining continuity with live message delivery. This ensures no messages are lost between history retrieval and subscription.
+
+<Aside data-type="note">
+By default, channels retain messages for 2 minutes without persistence enabled. For longer-term history, enable [persistence](/docs/storage-history/storage#all-message-persistence) on a channel namespace.
+</Aside>
+
+<Code>
+```javascript
+// Agent code
+import * as Ably from 'ably';
+
+const ably = new Ably.Realtime({
+  key: process.env.ABLY_API_KEY,
+  clientId: 'agent:assistant'
+});
+
+// Load persisted session state
+const channelName = await loadChannelName();
+const lastProcessedTimestamp = await loadLastProcessedTimestamp();
+
+// Use a channel in a namespace with persistence enabled
+// to access more than 2 minutes of message history
+const channel = ably.channels.get(channelName);
+
+// Subscribe to live messages (implicitly attaches the channel)
+await channel.subscribe('prompt', (message) => {
+  // Process the live message
+  processMessage(message);
+
+  // Persist the timestamp after successful processing
+  saveLastProcessedTimestamp(message.timestamp);
+});
+
+// Fetch history up until the point of attachment, starting from last checkpoint
+let page = await channel.history({
+  untilAttach: true,
+  start: lastProcessedTimestamp,
+  direction: 'forwards'
+});
+
+// Paginate through all missed messages
+while (page) {
+  for (const message of page.items) {
+    // Process the historical message
+    await processMessage(message);
+
+    // Persist the timestamp after successful processing
+    await saveLastProcessedTimestamp(message.timestamp);
+  }
+
+  // Move to next page if available
+  page = page.hasNext() ? await page.next() : null;
+}
+```
+</Code>
+
+<Aside data-type="note">
```

Comment:
This is quite an important note. Do you think it's worth being higher up in the page?